Doc/Tutorial/chapter_blast.tex

```diff
@@ -208,37 +208,24 @@ \subsection{Standalone NCBI BLAST+}
 On my computer this takes about six minutes - a good reason to save the output
 to a file so you can repeat any analysis as needed.
 
-From within Biopython we can use the NCBI BLASTX wrapper from the
-\verb|Bio.Blast.Applications| module to build the command line string,
-and run it:
+From within python we can use the \verb|subprocess| module to build the command line
+string, and run it:
 
 \begin{minted}{pycon}
->>> from Bio.Blast.Applications import NcbiblastxCommandline
->>> help(NcbiblastxCommandline)
-...
->>> blastx_cline = NcbiblastxCommandline(query="opuntia.fasta", db="nr", evalue=0.001,
-...                                      outfmt=5, out="opuntia.xml")
->>> blastx_cline
-NcbiblastxCommandline(cmd='blastx', out='opuntia.xml', outfmt=5, query='opuntia.fasta',
-db='nr', evalue=0.001)
->>> print(blastx_cline)
-blastx -out opuntia.xml -outfmt 5 -query opuntia.fasta -db nr -evalue 0.001
->>> stdout, stderr = blastx_cline()
+>>> import subprocess
+>>> cmd = "blastx -query opuntia.fasta -db nr -out opuntia.xml"
+>>> cmd += " -evalue 0.001 -outfmt 5"
+>>> subprocess.run(cmd, shell=True)
```

Comment:
Which OS have you tried this form on? If it works on Windows/macOS/Linux, then great.

Reply:
I have only tested on Linux, and don't have access to other OSes, thinking about it the only problem I could see is the `shell=True` having some weird behavior in Windows, but don't know enough about it to say for sure. Since it's such a simple example I would guess it would be fine.

Reply:
From memory, we're using a string for ``cmd`` here, so it should run on Windows - the difference may be in how the shell manifests (as a visible terminal window or not), and may also depend on if Python is run from a terminal or not.

Still, this is probably worth merging even without directly testing under Windows, but will have to wait a day or two as the current release is in progress.